### ISSUE TITLE
How can I keep treeview expand state after click other item?

### ISSUE THREAD

**Issue by ghking1** [doxygen/doxygen#7066]
![2019-06-18_215719](https://user-images.githubusercontent.com/8264304/59690780-8cc17280-9214-11e9-9ed5-fbfab6a3bb0c.png)
![2019-06-18_215759](https://user-images.githubusercontent.com/8264304/59690791-8fbc6300-9214-11e9-85dc-7736df9bb248.png)

when I click "Include" all "src" items are fold automaticlly, How can I keep "src" items keep expand when I click "Include".

**Comment by ghking1** [comment]
@albert-github hello, I have updated the information.

**Comment by ghking1** [comment]
@albert-github 
I have dived into "navtree.js", but has no answer, is It don't support currently? 
But I think It's really very important, consistently treeview UI can attract more user.

**Comment by gh-cyrillo** [comment]
Hi all,
I am new to doxygen, and came across the same "auto fold" treeview. I understand from current record, that enhancement, keeping the treeview expanded, is not implemented. Correct?

Is there by any chance a workaround for this?

**Comment by albert-github** [comment]
Unfortunately the project as supplied by @ghking1 is not accessible anymore (as it was on an external source)

@gh-cyrillo Which version of doxygen did you try?

When clicking on the small triangle in front of the text in the treeview the tree remains expanded, when clicking on a text the tree is collapsed except for the clicked item.

**Comment by gh-cyrillo** [comment]
Hi @albert-github,
I am currently using version 1.8.5.
I have not tried out the behaviour with newer version of doxygen. But I will and report back.

**Comment by DCDEIE** [comment]
Does a workaround exist for this issue? Is it possible to override the default functionality in navtree.js? I'm on Doxygen v1.15. Many thanks.

**Comment by albert-github** [comment]
To the best of my knowledge there isn't a workaround.
Maybe something would be possible in the js code.

**Comment by DCDEIE** [comment]
Yes I can imagine it's possible in the JS code. I haven't come across any workarounds searching the internet unfortunately.
